tools/generate_definitions.py

```diff
@@ -0,0 +1,375 @@
+"""Script to generate the definitions.json file from rippled source code."""
+
+import os
+import re
+import sys
+from pathlib import Path
+
+import httpx
+
+if len(sys.argv) != 2 and len(sys.argv) != 3:
+    print("Usage: python " + sys.argv[0] + " path/to/rippled [path/to/pipe/to]")
```

Comment:
```suggestion
    print("Usage: python " + sys.argv[0] + " path/to/rippled [optional/path/to/output/file]")
```

Reply:
The square brackets around an argument already indicates that it's optional. That's standard in CLI syntax.

Reply:
barring the use of the word `optional`, `path/to/output/file` is more informative than `path/to/pipe/to`. Especially for users who are not familiar with the pipe terminology.

Reply:
Fixed